step-templates/deploy-child-project.json

```diff
@@ -1,262 +1,273 @@
+  "Id": "2e6e2ee7-9ec5-45da-b26d-c9aea0d04e66",
```

Comment:
The IDs of the step must remain consistent. The changed ID here means Octopus will treat this as a new step rather than an update to the existing step.

Reply:
This is corrected in latest commit.